src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -90,11 +91,17 @@ public abstract class InteractiveEvaluator : IInteractiveEvaluator, ICurrentWork
             _commandsFactory = commandsFactory;
             _commands = commands;
 
-            var hostPath = interactiveHostPath;
-            _interactiveHost = new InteractiveHost(replType, hostPath, initialWorkingDirectory);
-            _interactiveHost.ProcessStarting += ProcessStarting;
-
+            // The following settings will apply when the REPL starts without .rsp file.
+            // They are discarded once the REPL is reset.
```

Comment:
"They are discarded by the default reset (which uses the .rsp file)."? (or something to that effect...it's not completely clear why these would be discarded)